src/probnum/quad/solvers/bayesian_quadrature.py

```diff
@@ -4,19 +4,19 @@
 
 import numpy as np
 
+from probnum.quad.solvers.policies import Policy, RandomPolicy
+from probnum.quad.solvers.stopping_criteria import (
+    BQStoppingCriterion,
```

Comment:
"IntegralVarianceTolerance", "MaxNevals", etc. are enough if they are grouped under probnum.quad.stopping_criteria. This way we don't need to duplicate the information contained in the package structure in the name. We will implement this change for `linalg` as well. The convention should then be to always import solver components as `stopping_criteria.MaxNevals`, for example.

Reply:
Ah, I see, I simply mirrored the naming of the current Linalg solvers API. I'll change this then.